pkg/scheduler/event_handler.go

```diff
@@ -58,6 +65,16 @@ func (s *Scheduler) addAllEventHandlers() {
 	s.eventRecorder = eventBroadcaster.NewRecorder(gclient.NewSchema(), corev1.EventSource{Component: "karmada-scheduler"})
 }
 
+func (s *Scheduler) resourceBindingEventFilter(obj interface{}) bool {
```

Comment:
Just a question: This change does not seem to have a relation with the current feature.

Reply:
Yeah, I have the same question, why is this present in this PR?

Reply:
> Yeah, I have the same question, why is this present in this PR?

For bindings created by dependencies distributor，they won't have poliy labels. Thus, no need to schedule.